### PR TITLE
package manifest.webapp and sw.js in war file [ng1]

### PR DESCRIPTION
**Pull request by ruddell**
- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [X] Tests are added where necessary
- [X] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angularjs/gulp/_copy.js [hunk 1]
```diff
@@ -84,7 +84,13 @@ function fonts() {
 }
 
 function common() {
-    return gulp.src([config.app + 'robots.txt', config.app + 'favicon.ico', config.app + '.htaccess'], { dot: true })
+    return gulp.src([
+        config.app + 'robots.txt',
+        config.app + 'favicon.ico',
+        config.app + '.htaccess',
+        config.app + 'manifest.webapp',
+        config.app + 'sw.js'
```
Comment (sendilkumarn) [inline generators/client/templates/angularjs/gulp/_copy.js]: can we comment this line ? We are not enabling it by default in other places.

Comment (deepu105) [inline generators/client/templates/angularjs/gulp/_copy.js]: Opps sorry I merged it too fast. will change it

Comment (deepu105) [inline generators/client/templates/angularjs/gulp/_copy.js]: done. @sendilkumarn can you add some documentation for this. Maybe  new page on PWA with instructions on how to enable it

Comment (sendilkumarn) [inline generators/client/templates/angularjs/gulp/_copy.js]: I have started that. I will commit it by this weekend.

**Pull request merged**